src/OpenTelemetry.AutoInstrumentation.Native/integration.cpp

```diff
@@ -10,171 +10,170 @@
 
 #include <unordered_map>
 
-#include "util.h"
 #include "logger.h"
+#include "util.h"
 
 namespace trace
 {
 
 std::mutex m_assemblyReferenceCacheMutex;
 std::unordered_map<WSTRING, std::unique_ptr<AssemblyReference>> m_assemblyReferenceCache;
 
-
-AssemblyReference::AssemblyReference(const WSTRING& str) :
-    name(GetNameFromAssemblyReferenceString(str)),
-    version(GetVersionFromAssemblyReferenceString(str)),
-    locale(GetLocaleFromAssemblyReferenceString(str)),
-    public_key(GetPublicKeyFromAssemblyReferenceString(str))
+AssemblyReference::AssemblyReference(const WSTRING& str)
+    : name(GetNameFromAssemblyReferenceString(str))
+    , version(GetVersionFromAssemblyReferenceString(str))
+    , locale(GetLocaleFromAssemblyReferenceString(str))
+    , public_key(GetPublicKeyFromAssemblyReferenceString(str))
 {
 }
 
 AssemblyReference* AssemblyReference::GetFromCache(const WSTRING& str)
 {
     std::lock_guard<std::mutex> guard(m_assemblyReferenceCacheMutex);
-    auto findRes = m_assemblyReferenceCache.find(str);
+    auto                        findRes = m_assemblyReferenceCache.find(str);
```

Comment:
This is odd, is there some misconfiguration?

Reply:
I take that back, it looks like this variable name is being aligned with the variable name on the previous line